src/plugins/dashboard/public/application/_dashboard_app.scss

```diff
@@ -66,4 +66,17 @@
   .dshUnsavedListingItem__actions {
     flex-direction: column;
   }
-}
\ No newline at end of file
+}
+
+// Temporary fix for two tone icons to make them monochrome
+.dshSolutionToolbar__editorContextMenu--dark {
+  .euiIcon path {
+    fill: $euiColorGhost;
+  }
+}
```

Comment:
@cqliu1 Is this something that needs to be fixed in EUI. If so can you point to the issue you were having that made you need to target `.euiIcon` directly and their SVG paths?

Reply:
Yeah, the dual color app icons don’t use the same color as the text on a button, so I had to manually adjust the icon color with CSS. I’ll file an issue in the EUI repo.